packages/web-components/README.md

```diff
@@ -1,176 +1,3 @@
-# \<agoric-wallet-connection>
```

Comment:
removing this is a breaking change. the commit should say `BREAKING CHANGE:` or at least have a `!` so it shows up as such in the CHANGELOG.

Note that I'm eager to evaluate alternatives to commit-based changelogs, but I think we should be consistent on that throughout the org or at least all the team's repos

Reply:
Done